### PR TITLE
Add qnm package

### PR DESCRIPTION
**Pull request by duetosymmetry**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] ~~If static libraries are linked in, the license of the static library is packaged.~~
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by duetosymmetry** [comment]
I confirm that I'm willing to be listed as maintainer

**Comment by duetosymmetry** [comment]
Note, I force pushed because I noticed that `openpathsampling` had a recipe on `upstream/master` when I had forked. I have rebased off of master since the `openpathsampling` directory has been removed.

File: recipes/qnm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 59d364dacbe5ea7b2b62844aad40f9351d987fe6e3c9c9b0d076574a8033d03f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```
Comment (npavlovikj) [inline recipes/qnm/meta.yaml]: I think this package can be noarch, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#architecture-independent-packages, so can you please add `noarch: python` in the `build` section here?

File: recipes/qnm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+requirements:
+  host:
```
Comment (marcelotrevisani) [inline recipes/qnm/meta.yaml]: I think you just need ``python`` and ``pip`` in ``host``, am I right?

File: recipes/qnm/meta.yaml (hunk 1 above)
Comment (duetosymmetry) [reply recipes/qnm/meta.yaml]: Thanks, @npavlovikj. Do I also want `--no-deps --ignore-installed` ?

File: recipes/qnm/meta.yaml (hunk 2 above)
Comment (duetosymmetry) [reply recipes/qnm/meta.yaml]: Maybe this question belonged here instead. Do I want `--no-deps --ignore-installed` on the pip command line?

File: recipes/qnm/meta.yaml (hunk 1 above)
Comment (duetosymmetry) [reply recipes/qnm/meta.yaml]: Added `noarch: python` and `--no-deps --ignore-installed` to the pip command line

File: recipes/qnm/meta.yaml (hunk 2 above)
Comment (duetosymmetry) [reply recipes/qnm/meta.yaml]: I've cut down the `host` section to just `pip` and `python`, and added `--no-deps --ignore-installed` to the pip command line

**Comment by duetosymmetry** [comment]
@npavlovikj @marcelotrevisani I've resolved your review suggestions above, do you think this commit is ready for merging?

**Comment by npavlovikj** [comment]
The recipe LGTM - thank you for contributing @duetosymmetry !

**Pull request merged**